lib/pushmi_pullyu/aip.rb

```diff
@@ -0,0 +1,66 @@
+require 'pushmi_pullyu'
+require 'bagit'
+require 'archive/tar/minitar'
+
+module PushmiPullyu::AIP
+  # Exceptions
+  class BagInvalid < StandardError; end
+  class NoContentFilename < StandardError; end
+  class FedoraFetchError < StandardError; end
+  class SolrFetchError < StandardError; end
+
+  def self.create(noid, skip_download: false, clean_work_directories: true)
```

Comment:
Minor tweak: give `skip_download` and `clean_work_directories` predicate names, like `should_skip_download`  or something